screen/edit-transaction/src/main/java/com/ivy/transaction/EditTransactionViewModel.kt

```diff
@@ -256,6 +284,18 @@ class EditTransactionViewModel @Inject constructor(
         return customExchangeRateState.value
     }
 
+    @Stable
```

Comment:
TIL! Nice 💯 Let's see if this will fix stability!

Reply:
@Vishwa-Raghavendra just add those to the baseline in the Compose Stability check. Your code is fine, I'll see how to fix this Stability thing properly some day 😀

Reply:
baseline doesn't work in windows the gradle  task fails 
even when I run it in wrapper mode

Reply:
Have a look in Telegram- I sent some instructions on how to fix it manually

Reply:
I'll update the check when I have time to exclude ViewModels